src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,59 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
+
+  //
+  // Removing 'garbage' from the response vector (when just *one* incoming contextElement
+  //
+  
+  if (upcr.contextElementResponseVector.size() != parseDataP->upcr.res.contextElementVector.size())
+  {
+    for (unsigned int ix = 0; ix < upcr.contextElementResponseVector.size() - 1; ++ix)
+    {
+      //
+      // 2 cases:
+      //
+      // o 1. 472+302:  keep 302
+      //   Out[0]: (472) request parameter is invalid/not allowed
+      //   Out[1]: (302) Found
+      //
+      // o 2. 472+404: keep 472
+      //   Out[0]: (472) request parameter is invalid/not allowed
+      //   Out[1]: (404) No context element found
+      //
+      if (upcr.contextElementResponseVector[ix]->statusCode.code == 472)
```

Comment:
Although it is ok to have 472, 404 and 302 in the comments, I think it should be better to have the corresponding Scc* in the code.